src/pages/singleOperator.tsx

```diff
@@ -106,9 +106,9 @@ export const getServerSideProps = async (
         throw new Error('Failed to retrieve operator cookie for single operator page');
     }
 
-    const operatorObject = JSON.parse(operatorCookie);
+    const operatorInfo = JSON.parse(operatorCookie);
 
-    const { nocCode } = operatorObject;
+    const { nocCode } = operatorInfo;
```

Comment:
Can we change lines 138-140 to copy this "don't mention object in the variable name" syntax?  😀

Reply:
Done!